Documentation/ceph-filesystem-crd.md

```diff
@@ -115,7 +115,14 @@ The pools allow all of the settings defined in the Pool CRD spec. For more detai
 
 * `metadataPool`: The settings used to create the filesystem metadata pool. Must use replication.
 * `dataPools`: The settings to create the filesystem data pools. If multiple pools are specified, Rook will add the pools to the filesystem. Assigning users or files to a pool is left as an exercise for the reader with the [CephFS documentation](http://docs.ceph.com/docs/master/cephfs/file-layouts/). The data pools can use replication or erasure coding. If erasure coding pools are specified, the cluster must be running with bluestore enabled on the OSDs.
-* `preserveFilesystemOnDelete`: If it is set to 'true' the filesystem will remain when the Filesystem CRD is deleted. This is a security measure to avoid accidental loss of data. It is set to 'false' by default. If not specified is also deemed as 'false'. This option replaces `preservePoolsOnDelete` which should no longer be set.
+* `preserveFilesystemOnDelete`: If it is set to 'true' the filesystem will remain when the
+  Filesystem CRD is deleted. This is a security measure to avoid loss of data if the CephFilesystem
+  Custom Resource is deleted accidentally. It is set to 'false' by default. If not specified is also
+  deemed as 'false'. This option replaces `preservePoolsOnDelete` which should no longer be set.
+* (DEPRECATED) `preservePoolsOnDelete`: This option is replaced by the above
```

Comment:
Since we handle the behavior automatically and there is no breaking change, the previous description seems like it is still valid and we shouldn't need to mention it as a deprecated feature.

Reply:
I think it's useful to mark the old option as deprecated because it no longer applies in its previous form. Kubernetes deprecates API elements long before they are removed.